src/signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/DetailPanel.tsx

```diff
@@ -1,312 +1,183 @@
 // SPDX-License-Identifier: MPL-2.0
-// Copyright (C) 2021 - 2022 Gemeente Amsterdam, Vereniging van Nederlandse Gemeenten
-import { useCallback, useState, useContext, useRef, useEffect } from 'react'
-import type { KeyboardEvent, ChangeEvent, FC } from 'react'
+import type { FC } from 'react'
+// Copyright (C) 2021 - 2023 Gemeente Amsterdam, Vereniging van Nederlandse Gemeenten
```

Comment:
Shouldn't this be `2024`? Or, even better, shouldn't we remove the year entirely? See also [Copyright Notices in Open Source Software Projects](https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects)

Reply:
I think we can skip this whole copyright thing in the next version

Reply:
I don't think we should, it's very useful for open source software. But maybe we can simplify it.